app/views/admin/collections/_form.html.erb

```diff
@@ -109,3 +109,15 @@
     <%= f.submit class: "usa-button" %>
   </p>
 <% end %>
+<script>
+  $('input,textarea,select,radio').on('blur',function(e){
```

Comment:
I could see this method being generalized into a function, and referenced via views that need it.